psd-web/app/validators/uuid_validator.rb

```diff
@@ -0,0 +1,7 @@
+class UuidValidator < ActiveModel::EachValidator
```

Comment:
```suggestion
class UUIDValidator < ActiveModel::EachValidator
```
and in `config/initializers/inflections.rb`add:
```ruby
inflect.acronym 'UUID'
```

Reply:
Nice!